test/components/New_Order_spec.js

```diff
@@ -28,35 +28,69 @@ describe('New Order', () => {
     expect(options.length).to.equal(masterItems.length);
   });
 
-  it('adds an item to the list each time', () => {
+  it('adds an entry to the list each time', () => {
     const item = { id: '1', name: 'food' };
     const component = renderIntoDocument(NewOrder({ masterItems: [item], loadItems }));
-    const addItemBtn = findRenderedDOMComponentWithClass(component, 'add-item');
+    const addEntryBtn = findRenderedDOMComponentWithClass(component, 'add-entry');
+    const addCommentFld = findRenderedDOMComponentWithClass(component, 'add-comment');
 
-    Simulate.click(addItemBtn);
-    Simulate.click(addItemBtn);
+    Simulate.click(addEntryBtn);
 
-    const addedItems = scryRenderedDOMComponentsWithClass(component, 'added-item');
-    expect(addedItems.length).to.equal(2);
-    expect(addedItems[0].textContent).to.contain('food');
+    const entries = scryRenderedDOMComponentsWithClass(component, 'entries');
+    const entryName = scryRenderedDOMComponentsWithClass(component, 'entry-name');
+    expect(entries.length).to.equal(1);
+    expect(entryName[0].textContent).to.equal('food');
   });
 
-  it('calls adds order on submit', () => {
+  it('adds a new comment for each entry', () => {
+    const item = { id: '1', name: 'food' };
+    const component = renderIntoDocument(NewOrder({ masterItems: [item], loadItems }));
+    const addEntryBtn = findRenderedDOMComponentWithClass(component, 'add-entry');
+    const addCommentFld = findRenderedDOMComponentWithClass(component, 'add-comment');
+
+    Simulate.change(addCommentFld, { target: { value: 'comment' } });
+    Simulate.click(addEntryBtn);
+    Simulate.change(addCommentFld, { target: { value: 'new comment' } });
+    Simulate.click(addEntryBtn);
+
+    const entryComment = scryRenderedDOMComponentsWithClass(component, 'entry-comment');
+
+    expect(entryComment[0].textContent).to.equal('comment');
+    expect(entryComment[1].textContent).to.equal('new comment');
+  });
+
+  xit('clears the comment after adding an entry', () => {
```

Comment:
This test looks fine, I think we just need to add `value: this.state.comment` to the input field element props for it to pass.